src/index.ts

```diff
@@ -0,0 +1,346 @@
+import cluster from 'cluster'
+import nano from 'nano'
+import fetch from 'node-fetch'
+import { cpus } from 'os'
+
+import { matchJson } from './match-json'
+
+export interface UserInfo {
+  couchUri: string
+  name: string
+  password: string
+  databases?: string[]
+}
+
+export class ServerUtilError extends Error {
+  response: object
+
+  constructor(message: string, response: object) {
+    super(message)
+    this.response = response
+  }
+}
+
+export async function createRegularUser(userInfo: UserInfo): Promise<void> {
+  const user = {
+    name: userInfo.name,
+    password: userInfo.password,
+    roles: [],
+    type: 'user'
+  }
+  const completeUri = `${userInfo.couchUri}/_users/org.couchdb.user:${userInfo.name}`
+  const response = await fetch(completeUri, {
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    method: 'PUT',
+    body: JSON.stringify(user)
+  })
+  if (response.ok === false) {
+    const errorResponse = await response.json()
```

Comment:
Continue on "user already exist" type of error responses. This way we can continue safely and not throw.